src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs

```diff
@@ -46,8 +46,8 @@ public class Package
         private List<string> _privileges;
         private int _installedTime;
 
-        private Dictionary<IntPtr, Interop.PackageManager.PackageManagerSizeInfoCallback> _packageManagerSizeInfoCallbackDict = new Dictionary<IntPtr, Interop.PackageManager.PackageManagerSizeInfoCallback>();
-        private int _callbackId = 0;
+        private static Dictionary<IntPtr, Interop.PackageManager.PackageManagerSizeInfoCallback> _packageManagerSizeInfoCallbackDict = new Dictionary<IntPtr, Interop.PackageManager.PackageManagerSizeInfoCallback>();
```

Comment:
Please use ConcurrentDictionary instead of Dictionary for thread safe

Reply:
Please ignore this comment. 
The implementation already used lock for thread safe.